common/config/azure-pipelines/templates/core-build.yaml

```diff
@@ -34,7 +34,23 @@ steps:
     condition: and(succeeded(), eq(variables['Build.Reason'], 'PullRequest'), eq(variables['Agent.OS'], 'Darwin'))
   - script: npm run build:ios
     workingDirectory: "test-apps/display-test-app"
-    displayName: "Build iOS display-test-app"
+    displayName: "Build iOS display-test-app typescript"
+    condition: and(succeeded(), eq(variables['Build.Reason'], 'PullRequest'), eq(variables['Agent.OS'], 'Darwin'))
+  - task: InstallAppleProvisioningProfile@1
```

Comment:
Can we make this an addition step within the fast-ci.yaml instead of this one? This one is shared by many different pipelines and we don't want secrets in a pipeline that can be run on a fork (which this one can be via the ci.yaml file).

Reply:
Yes, that would be fine with me. I might need help setting that up.

Reply:
@calebmshafer I added a `buildIos` parameter to ci-core and core-build that is then only set to true in fast-ci. This works but I could potentially move the `npm run build:ios-app` script into ci-core if you think that would be better?

Reply:
@calebmshafer @aruniverse Can we possibly move forward with this PR?

Reply:
pipeline changes look ok to me, will approve. leaving comment open for caleb to resolve next week and for him to approve too